docs/DeveloperGuide.md

```diff
@@ -486,367 +498,250 @@ Hence, any modification to an employee after SudoHR is initialized from storage
-| `* * *`  | HR personnel                           | list all departments an employee is in                         |                                                                            |
-| `* * *`  | HR personnel                           | list all employees in a department                             | view manpower size of a department                                         |
-
-
-*{More to be added}*
+| Priority | As a …​      | I want to …​                                                   | So that I can…​                                                            |
+| -------- | ------------ | -------------------------------------------------------------- | -------------------------------------------------------------------------- |
+| `* * *`  | new user     | see usage instructions                                         | refer to instructions when I forget how to use the App                     |
+| `* * *`  | HR personnel | add a new employee                                             | ensure consolidation of information when an employee is hired              |
+| `* * *`  | HR personnel | edit a new employee                                            | ensure consolidation of information when an employee has new details       |
+| `* * *`  | HR personnel | delete an employee                                             | ensure consolidation of information when an employee left the company      |
+| `* * *`  | HR personnel | find an employee by name                                       | locate details of employees without having to go through the entire list   |
+| `* *`    | HR personnel | hide private contact details                                   | minimize chance of someone else seeing them by accident                    |
+| `* * *`  | HR personnel | add an employee’s leave to SudoHR                              | ensure consolidation of information                                        |
+| `* * *`  | HR personnel | remove an employee’s leave for SudoHR                          | ensure consolidation of information                                        |
+| `* * *`  | HR personnel | view all leaves an employee has applied for                    | access an employee's availability easily                                   |
+| `* * *`  | HR personnel | view all employees on leave today                              | know today's headcount                                                     |
+| `* * *`  | HR personnel | view all leaves applied for a given day                        | better plan company events                                                 |
+| `* * *`  | HR personnel | view all leaves applied for a given day for a given department | better plan depeartment events                                             |
```

Comment:
It may be better if write this as view all employees not on leave for a give day for a department